### PR TITLE
Integrate e2e tests with MSBuild

### PR DESCRIPTION
**Pull request by Xiaohongt**

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@Xiaohongt__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft Open Technologies, Inc. and real humans are currently evaluating your PR.</p>

TTYL, MSOTBOT;

File: Build/build.msbuild [hunk 1]
```diff
@@ -40,8 +51,12 @@
 
   <Target Name="Test" DependsOnTargets="Build">
     <Exec Command="$(OutDir)\signalrclienttests.exe --gtest_output=xml:$(OutDir)\test_results.xml" />
+    <ExecAsync Executable="$(SolutionDir)test\signalrclient-testhost\bin\$(Configuration)\signalrclient-testhost.exe" Arguments="180000" />
```
Comment (moozzyk) [inline Build/build.msbuild]: Make this timeout smaller (like a minute or so - currently running the tests takes less than a second)

File: Build/build.msbuild [hunk 2]
```diff
@@ -40,8 +51,12 @@
 
   <Target Name="Test" DependsOnTargets="Build">
     <Exec Command="$(OutDir)\signalrclienttests.exe --gtest_output=xml:$(OutDir)\test_results.xml" />
+    <ExecAsync Executable="$(SolutionDir)test\signalrclient-testhost\bin\$(Configuration)\signalrclient-testhost.exe" Arguments="180000" />    
+    <Exec Command="$(OutDir)\signalrclient-e2e-tests.exe --gtest_output=xml:$(OutDir)\e2e_test_results.xml" />
+    <Exec Command="taskkill /IM signalrclient-testhost.exe /F" ContinueOnError="true" />
   </Target>
 
+
```
Comment (moozzyk) [inline Build/build.msbuild]: Nit: remove this empty line

**Comment by moozzyk** [comment]
:ship::it: after you fix the nits.

**Pull request merged**